src/scripts/script.js

```diff
@@ -0,0 +1,342 @@
+const cardDrop = document.querySelector("#card-riddance-modal");
+const previewModal = document.querySelector("#preview-modal");
+const previewImage = previewModal.querySelector(".modal__preview-image");
+const previewTitle = previewModal.querySelector(".modal__preview-title");
+const newPostButton = document.querySelector(".profile__post-button");
+const editCardModal = document.querySelector("#add-card-modal");
+const editProfileModal = document.querySelector("#edit-modal");
+const profileName = document.querySelector(".profile__name");
+const profileTitle = document.querySelector(".profile__title");
+const profileInputName = document.querySelector("#name");
+const profileInputTitle = document.querySelector("#description");
+const profileEditButton = document.querySelector(".profile__edit-button");
+const editForm = document.forms["edit-profile"];
+const cardForm = editCardModal.querySelector(".modal__form");
+const cardTemplate = document.querySelector("#card-template");
+const cardSection = document.querySelector(".cards__list");
+const cardNameInput = document.querySelector("#add-card-name");
+const cardLinkInput = document.querySelector("#add-card-link");
+
+let cardToDelete = null;
+let cardIdToDelete = null;
+
+function getCard(data) {
+  const card = cardTemplate.content.querySelector(".card").cloneNode(true);
+  const cardName = card.querySelector(".card__title");
+  const cardImage = card.querySelector(".card__image");
+  const cardLikeIcon = card.querySelector(".card__like-icon");
+  const deleteButton = card.querySelector(".card__delete");
+
+  cardImage.src = data.link;
+  cardImage.alt = data.name;
+  cardName.textContent = data.name;
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+  cardLikeIcon.addEventListener("click", () => {
+    console.log(data);
+    if (cardLikeIcon.classList.contains("card__like-icon_active")) {
+      // DISLIKE
+      api
+        .deleteCardLike(data._id)
+        .then(() => {
+          cardLikeIcon.classList.remove("card__like-icon_active");
+        })
+        .catch((err) => {
+          cardLikeIcon.classList.remove("card__like-icon_active");
+          console.error("Error removing like:", err);
+        });
+    } else {
+      // LIKE
+      api
+        .addCardLike(data._id)
+        .then(() => {
+          cardLikeIcon.classList.add("card__like-icon_active");
+        })
+        .catch((err) => {
+          cardLikeIcon.classList.add("card__like-icon_active");
+          console.error("Error adding like:", err);
+        });
+    }
+  });
+
+=======
+=======
+  // Add like toggle functionality
+  cardLikeIcon.addEventListener("click", () => {
+    cardLikeIcon.classList.toggle("card__like-icon_active");
+  });
+
+>>>>>>> 3494d28 (fixed errors)
+  // When trash icon is clicked, open confirmation and store card info
+>>>>>>> parent of 3494d28 (fixed errors)
+  deleteButton.addEventListener("click", () => {
+    cardDrop.classList.add("card-riddance__opened");
```

Comment:
`card-riddance__opened` is not needed in the project because all modals should use `modal_opened` and the `openModal()` or `closeModal()` functions for opening or closing